tests/tensorflow/README.md

```diff
@@ -11,10 +13,9 @@ In this folder, there are test files available to test if the nncf module is ins
 If you want to do pre-commit test, you can enter the command as below.
 ```
 pytest tests/common tests/tensorflow \
-       --junitxml nncf-tests.xml \
-       --data-dir tests/tensorflow/data
```

Comment:
`pytest` has no `--data-dir` argument

Reply:
Please find the correct command to run the pre-commit test: https://github.com/openvinotoolkit/nncf/blob/63d0b543bd60cd49061383310ed562a19744004a/Makefile#L79

I would recommend adding to the documentation a generic way to run a test before commit via make:
```
make install-tensorflow-test
make test-tensorflow
```